src/rp2350/boot_stage2/include/boot_stage2/config.h

```diff
@@ -11,6 +11,9 @@
 
 #include "pico.h"
 
+// PICO_CONFIG: PICO_FLASH_SPI_CLKDIV, Clock divider from clk_sys to use for serial flash communications in boot stage 2. On RP2040 this must be a multiple of 2, type=int, default=varies; often specified in board header, advanced=true, group=boot_stage2
```

Comment:
This file is in `src/rp2350/...` so is RP2350-specific, so the whole "On RP2040 this must be a multiple of 2" statement can probably be removed?